boilerplate/app/navigation/root-navigator.ts

```diff
@@ -1,5 +1,7 @@
 import { createStackNavigator } from "react-navigation"
 import { PrimaryNavigator } from "./primary-navigator"
+import {
+} from "../screens" // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
I thought that too, but I think this is where the screen generator will insert references to the new screen.

Reply:
Bryan is correct. Because screens are all being imported from the same place now, in order to be able to patch the file, I needed the empty import to be pre-existing. Previously, we were just adding a whole new line for each screen.

Reply:
Oh, gotcha!